src/Categories/Tactic/Category.agda

```diff
@@ -19,166 +19,165 @@ open import Data.List    as List    using (List; _∷_; [])
-
-_==_ = primQNameEquality
-{-# INLINE _==_ #-}
-
-getArgs : Term → Maybe (Term × Term)
-getArgs (def _ xs) = go xs
-  where
-  go : List (Arg Term) → Maybe (Term × Term)
-  go (vArg x ∷ vArg y ∷ []) = just (x , y)
-  go (x ∷ xs)               = go xs
-  go _                      = nothing
-getArgs _ = nothing
-
---------------------------------------------------------------------------------
--- Getting Category Names
---------------------------------------------------------------------------------
-
-record CategoryNames : Set where
-  field
-    is-∘ : Name → Bool
-    is-id : Name → Bool
-
-buildMatcher : Name → Maybe Name → Name → Bool
-buildMatcher n nothing  x = n == x
-buildMatcher n (just m) x = n == x ∨ m == x
-
-findCategoryNames : Term → TC CategoryNames
-findCategoryNames cat = do
-  ∘-altName ← normalise (def (quote Category._∘_) (3 ⋯⟅∷⟆ cat ⟨∷⟩ []))
-  id-altName ← normalise (def (quote Category.id) (3 ⋯⟅∷⟆ cat ⟨∷⟩ []))
-  returnTC record
-    { is-∘ = buildMatcher (quote Category._∘_) (getName ∘-altName)
-    ; is-id = buildMatcher (quote Category.id) (getName id-altName)
-    }
-
---------------------------------------------------------------------------------
--- Constructing an Expr
---------------------------------------------------------------------------------
-
-″id″ : Term
-″id″ = quote id′ ⟨ con ⟩ []
-
-″[_↑]″ : Term → Term
-″[ t ↑]″ = quote [_↑] ⟨ con ⟩ (t ⟨∷⟩ [])
-
-module _ (names : CategoryNames) where
-
-  open CategoryNames names
-
-  mutual
-    ″∘″ : List (Arg Term) → Term
-    ″∘″ (x ⟨∷⟩ y ⟨∷⟩ xs) = quote _∘′_ ⟨ con ⟩ buildExpr x ⟨∷⟩ buildExpr y ⟨∷⟩ []
-    ″∘″ (x ∷ xs) = ″∘″ xs
-    ″∘″ _ = unknown
-
-    buildExpr : Term → Term
-    buildExpr t@(def n xs) =
-      if (is-∘ n)
-        then ″∘″ xs
-      else if (is-id n)
-        then ″id″
-      else
-        ″[ t ↑]″
-    buildExpr t@(con n xs) =
-      if (is-∘ n)
-        then ″∘″ xs
-      else if (is-id n)
-        then ″id″
-      else
-        ″[ t ↑]″
-    buildExpr t = ″[ t ↑]″
-
---------------------------------------------------------------------------------
--- Constructing the Solution
---------------------------------------------------------------------------------
-
-constructSoln : Term → CategoryNames → Term → Term → Term
-constructSoln cat names lhs rhs =
-  quote Category.Equiv.trans ⟨ def ⟩ 3 ⋯⟅∷⟆ cat ⟨∷⟩
-    (quote Category.Equiv.sym ⟨ def ⟩ 3 ⋯⟅∷⟆ cat ⟨∷⟩
-      (quote preserves-≈ ⟨ def ⟩ 3 ⋯⟅∷⟆ cat ⟨∷⟩ buildExpr names lhs ⟨∷⟩ []) ⟨∷⟩ [])
-    ⟨∷⟩
-    (quote preserves-≈ ⟨ def ⟩ 3 ⋯⟅∷⟆ cat ⟨∷⟩ buildExpr names rhs ⟨∷⟩ [])
-    ⟨∷⟩ []
-
-solve-macro : Term → Term → TC _
-solve-macro mon hole = do
-  hole′ ← inferType hole >>= normalise
-  names ← findCategoryNames mon
-  just (lhs , rhs) ← returnTC (getArgs hole′)
-    where nothing → typeError (termErr hole′ ∷ [])
-  let soln = constructSoln mon names lhs rhs
-  unify hole soln
-
-macro
-  solve : Term → Term → TC _
-  solve = solve-macro
-
+-- open import Reflection.Argument
```

Comment:
Are the edits to this file meant to be in this PR?

Reply:
No, I think we just forgot to unstage this temporary fix, since Reflection changed in the latest version of agda-stdlib, thank you for the heads up!